flink-connectors/flink-connector-rabbitmq/src/test/java/org/apache/flink/streaming/connectors/rabbitmq/RMQSourceTest.java

```diff
@@ -419,6 +426,48 @@ protected Connection setupConnection() throws Exception {
         Mockito.verify(channel, Mockito.times(0)).basicQos(anyInt());
     }
 
+    @Test
+    public void testDeliveryTimeout() throws Exception {
+        source.autoAck = false;
```

Comment:
I think with the current setup we have for the RabbitMQ source it is very tough to test the added functionality. We would need to access the underlying `queue` in the Consumer to mock the behavior. 
My suggestion would be to change this test slightly to ensure `consumer.nextDelivery()` is never called.

Reply:
Thanks. I've cherry-picked the integration tests you implemented. Looks great (I've just removed a duplicated dependency - 12ea915).
I've changed (b86d622) the deliveryTimeout test (it now basically just checks whether the correct method was called).

Reply:
Thanks, and I am a bit sorry for the missed styling check but you've already fixed it 👍